src/index.js

```diff
@@ -286,8 +286,8 @@ function EvolvClient(opts) {
               }
             });
 
-
-            context.set('experiments.confirmations', contextConfirmations.concat(confirmations));
+            const contextConfirmationsKey = store.isInternalUser() ? 'experiments.confirmationsinternal' : 'experiments.confirmations';
```

Comment:
can we do experiments.confirmationsInternal?

Reply:
Yea, we probably can. I was looking around to see if we were doing any other camel casing, but the closest I saw in the config file was `timestamp`, so I figured I'd go with the ticket, which was apparently a typo 😅